docs/source/optional_plugins/golang.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Golang Plugin
+=============
+
+This optional plugin enables Avocado to work with tests originally
+written using the `Go testing package <https://golang.org/pkg/testing/>`_.
+
+To install the Golang plugin from pip, use::
+
+    $ sudo pip install avocado-framework-plugin-golang
+
+After installed, you can list/run Golang tests providing the package name::
+
+    ~$ avocado list golang.org/x/text/unicode/norm
```

Comment:
While running with an invalid reference:

```
$ avocado list /path/doesnot/exist
Test discovery plugin <avocado_golang.GolangLoader object at 0x7f4a6be46e90> failed: 'NoneType' object has no attribute 'endswith'
Unable to resolve reference(s) '/path/doesnot/exist' with plugins(s) 'file', 'golang', 'yaml_testsuite', 'robot', 'external', try running 'avocado list -V /path/doesnot/exist' to see the details.
```

The first line is most probably not a good idea.

Reply:
Ops :)